### ISSUE TITLE
fusiondirectory never starts as it's waiting for argonaut to finish init'ing

### ISSUE THREAD

**Issue by frebib** [tiredofit/docker-fusiondirectory#27]
The logs contain this
```
[cont-init.d] 30-argonaut: executing...
[DEBUG] ** [argonaut] Checking to see if service has initialized
[cont-init.d] 30-argonaut: exited 0.
[cont-init.d] 40-fusiondirectory: executing...
[DEBUG] ** [fusiondirectory] Checking to see if service has initialized
[DEBUG] ** [fusiondirectory] Looking for existence of /tmp/state/30-argonaut-init
[DEBUG] ** [fusiondirectory] Looking for existence of /tmp/state/30-argonaut-init
... (that last line repeats forever)
```
as it's waiting for `/tmp/state/30-argonaut-init`. Argonaut is disabled and `liftoff` is never called, that file is never created.

**Comment by frebib** [comment]
The fix should be as simple as
```
diff --git a/install/etc/cont-init.d/30-argonaut b/install/etc/cont-init.d/30-argonaut
index 282a5ad..1172291 100755
--- a/install/etc/cont-init.d/30-argonaut
+++ b/install/etc/cont-init.d/30-argonaut
@@ -13,3 +13,4 @@ else
   service_stop 30-argonaut
 fi

+liftoff
```
I can PR, but you'll probably beat me to fixing it :)

**Comment by tiredofit** [comment]
Gosh am I hopelessly sloppy these past few days, so I apologize. Grateful for your eagle eyes. I have a fix built but waiting for Docker Hub to finish off its build cycle on a couple dozen images before I can send this one in. An older release will work for the time being. Soon as Docker finishes I'll get this right out.

**Comment by tiredofit** [comment]
So sorry for the delay. A new build has been queued up with the Docker Hub, still waiting for all dependent images to build and then it will work as per normal again.

**Comment by frebib** [comment]
Seems to be fixed as of 3861e7ea750bcf11aea4b3663c1e9469af4f3659
Thanks!
